requirements_dev.txt

```diff
@@ -1,2 +1,3 @@
 pytest
 perf >= 0.8.0
+wcwidth
```

Comment:
`setup.py` don't install this.
My point is getting ready-to-go enviroment after `pip install -r`

I think we must add all dependencies here.